frontend/appflowy_flutter/integration_test/board/board_add_row_test.dart

```diff
@@ -68,12 +67,11 @@ void main() {
 
       final findLastCard = find.descendant(
         of: find.byType(AppFlowyGroupCard),
-        matching: find.byType(FlowyText),
+        matching: find.byType(Text),
```

Comment:
Are we use FlowyText for all the text?

Reply:
I change it to Text because it's easier to specify a `TextStyle` in the card cell style, rather than passing in fontsize, font color, overflow behavior, font weight, etc. all separately.